features/step_definitions/aids_for_devops_steps.rb

```diff
@@ -27,5 +27,6 @@
 end
 
 When(/^I start to create a form$/) do
+  fill_in 'Please name your research session', with: 'Any old pony'
```

Comment:
We should use a realistic example for context for the next maintainers

Reply:
It means we don't care about the value in this context, just that one exists – perhaps that could be more explicit.